app/classifier/tasks/drawing/summary.cjsx

```diff
@@ -1,4 +1,6 @@
 React = require 'react'
+strip = require 'strip-markdown'
+remark = (require 'remark').use(strip)
```

Comment:
I had a play around with passing the `zero` option to `markdown-it` before parsing the question, but that just passes back the raw markdown text. Not hugely useful.

Seems a pain to require browsers to download `remark` just for this one line of code. Long term, I'd like to look at extending `markdownz` to return the plain text content of a markdown string.